float64 pos_x
float64 pos_y
string context
int32 id

